middleware/lang-resolver.js

```diff
@@ -45,7 +48,7 @@ module.exports = function langResolver (req, res, next) {
 
     // redirect only when the resolved lang exists and it is different
     if (rlang && rlang !== req.query.lang) {
-      const parsedUrl = parseUrl(req)
+      const parsedUrl = parseurl(req)
```

Comment:
Why not `url.parse(req.url)` directly?

Reply:
well, I like parseurl (https://www.npmjs.com/package/parseurl) because it reuses the req._parsedUrl and it has better performance. But @zeke recommended me to avoid adding extra dependencies. So, I did remove it from my original pull but keeping parseurl(req) in case of we use it in the future.